spec/rails_helper.rb

```diff
@@ -53,6 +29,17 @@
     DatabaseCleaner.strategy =
       example.metadata[:js] ? :truncation : :transaction
     DatabaseCleaner.start
+    OmniAuth.config.test_mode = true
+    facebook = OmniAuth::AuthHash.new({
+    provider: "facebook",
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.